coc/client.py

```diff
@@ -147,8 +126,8 @@ def __init__(self, tokens=None, loop=None, email=None, password=None, update_tok
         log.info('Clash of Clans client logging out...')
         await self.http.close()
 
-    async def on_token_reset(self, new_token):
-        """Event: called when the client's token is reset.
+    async def on_key_reset(self, new_key):
+        """Event: called when the client's key is reset.
```

Comment:
since we're dealing with multiple keys should this line look something like "called when one of the clients keys..." or "called when a client key..."